hack/build-e2e-images.sh

```diff
@@ -5,6 +5,7 @@
 #
 # Usage:
 #   ./build-e2e-images.sh
+set -xe
```

Comment:
Without this, docker image builds could fail (e.g. because launchpad has a bad time again) silently and cause image pull error down the line.